webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/comment/list/ActivityCommentsDrawer.vue

```diff
@@ -195,15 +215,25 @@ export default {
             this.drawerOpened = true;
             this.scrollOnOpen = !options.editComment && !options.noAuitomaticScroll;
             this.retrieveComments(true, true);
-            this.$nextTick().then(() => {
-              if (this.$refs.activityCommentsDrawer) {
-                this.$refs.activityCommentsDrawer.open();
-              }
-            });
+            this.$nextTick().then(() => this.openDrawer());
           }
         });
       }
     },
+    displaySelectedActivityComment(commentId, activityId) {
```

Comment:
comment highlighting already work (you can try [this link](https://community.exoplatform.com/portal/dw/activity?id=397094#comment-comment397184) by example)
Why adding a new method to highlight it and display comments drawer ?

Reply:
It is not for highlight comment, it is just to open comment drawer from url, since we need to get activity by Id

Reply:
Updated!